app/views/charge.njk

```diff
@@ -492,22 +492,11 @@
     </aside>
   </div>
 </div>
-  <script id="govuk-script-charge">
-    window.chargeId = "{{ id }}"
-  </script>
+
 {% endblock %}
 
 {% block bodyEnd %}
-  {% if service.collectBillingAddress %}
-  <script type="text/javascript" src="/public/location-autocomplete.min.js"></script>
-  <script type="text/javascript">
-    openregisterLocationPicker({
-      selectElement: document.getElementById('address-country'),
-      url: '/public/countries-autocomplete-graph.json',
-      autoselect: true,
-      displayMenu: 'overlay'
-    })
-  </script>
-  {% endif %}
+  <input id="collect-billing-address" name="collectBillingAddress" type="hidden" value="{{ service.collectBillingAddress }}"/>
```

Comment:
As discussed we might be able to keep very similar behaviour by separating and templating in the initialise method into `scripts.njk` -- this would mean there are no different behaviours introduced. 

It would look something like: 
```js
{% if service.collectBillingAddress %}
window.payScripts.helpers.loadLocationAutocompleteScript()
{% endif %}
```

Reply:
👍 Updated